packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# 49.12.0 - September 30, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Show progress bar when installing dependencies for Function Create ([PR #2563](https://github.com/forcedotcom/salesforcedx-vscode/pull/2563))
+
+- Add feature flag for Functions ([PR #2572](https://github.com/forcedotcom/salesforcedx-vscode/pull/2572))
+
+- Show message if no default org is set when starting Function ([PR #2560](https://github.com/forcedotcom/salesforcedx-vscode/pull/2560))
```

Comment:
@jag-sfdc @ekapner are we adding Functions related things to docs yet? If yes, we can build out these entries more

Reply:
FYI @danyusf 

Per @jag-sfdc's doc, you need to go into Settings to enable the extension - but unsure if we want to announce its existence to the entire world.  Functions is having a closed pilot on Oct 19 - so I'm leaning toward that we skip documenting anything here - but let Dan document how to enable the extension in his pilot docs.

Reply:
Once Functions goes to public beta, we can make an announcement about the extension.

Reply:
Sounds good, I'll remove these entries for now

Reply:
Sorry missed this conversation. Yes, we shouldn't make an announcement yet.